patterns/product-details-listing.php

```diff
@@ -4,56 +4,41 @@
-			<!-- /wp:paragraph -->
+<!-- wp:woocommerce/single-product {"productId":<?php echo esc_attr( $product_id ); ?>} -->
+<div class="wp-block-woocommerce-single-product">
+	<!-- wp:columns -->
+	<div class="wp-block-columns">
+		<!-- wp:column -->
+		<div class="wp-block-column">
+			<!-- wp:post-featured-image {"height":"490px"} /-->
 
-			<!-- wp:paragraph {"style":{"typography":{"fontSize":"28px","fontWeight":"400"},"layout":{"selfStretch":"fit","flexSize":null},"color":{"text":"#d3d3d3ff"}}} -->
-			<p class="has-text-color" style="color:#d3d3d3ff;font-size:28px;font-weight:400;"><s>$47.49</s></p>
-			<!-- /wp:paragraph -->
-		</div>
-		<!-- /wp:group -->
+			<!-- wp:group {"style":{"spacing":{"blockGap":"8px"}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"center"}} -->
+			<div class="wp-block-group">
+				<!-- wp:woocommerce/product-rating-stars {"isDescendentOfSingleProductBlock":true,"textColor":"luminous-vivid-amber","style":{"spacing":{"margin":{"right":"5px"}}}} /-->
+				<!-- wp:woocommerce/product-average-rating {"style":{"typography":{"fontWeight":"600","fontSize":"14px"}}} /-->
+				<!-- wp:woocommerce/product-rating-counter {"isDescendentOfSingleProductBlock":true,"style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}},"typography":{"fontSize":"14px"}}} /-->
+			</div>
+			<!-- /wp:group -->
 
-		<!-- wp:buttons {"style":{"spacing":{"blockGap":"0px"}}} -->
-		<div class="wp-block-buttons is-content-justification-center">
-			<!-- wp:button {"backgroundColor":"black","textColor":"white","className":"is-style-fill","fontSize":"medium","style":{"spacing":{"padding":{"left":"94px","right":"94px","top":"20px","bottom":"20px"}}}} -->
-			<div class="wp-block-button has-custom-font-size is-style-fill has-medium-font-size"><a class="wp-block-button__link has-white-color has-black-background-color has-text-color has-background wp-element-button" style="padding-right:94px;padding-left:94px;padding-top:20px;padding-bottom:20px;"><strong>Add to cart</strong></a></div>
-			<!-- /wp:button -->
+			<!-- wp:post-title {"textAlign":"center","isLink":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"700"},"elements":{"link":{"color":{"text":"#000001"}}},"color":{"text":"#000001"}},"fontSize":"x-large","__woocommerceNamespace":"woocommerce/product-query/product-title"} /-->
+			<!-- wp:woocommerce/product-price {"textAlign":"center","isDescendentOfSingleProductBlock":true,"style":{"typography":{"fontSize":"28px","fontStyle":"normal","fontWeight":"700"}}} /-->
+			<!-- wp:woocommerce/product-button {"textAlign":"center","isDescendentOfSingleProductBlock":true,"style":{"color":{"background":"#000001","text":"#fffff1"},"typography":{"fontSize":"18px","fontStyle":"normal","fontWeight":"600"},"spacing":{"padding":{"top":"20px","bottom":"20px","right":"94px","left":"94px"},"margin":{"top":"15px"}}}} /-->
```

Comment:
After investigating https://github.com/woocommerce/woocommerce/issues/39358, I think we should remove the styles from this button, otherwise we will end up with an issue like https://github.com/woocommerce/woocommerce/issues/39358 again. The reason is that some of these styles (ie: color, spacing, etc.) are only enabled in the feature plugin and not in WC core:

https://github.com/woocommerce/woocommerce-blocks/blob/b4ba06a6242cbb6a64d2ec554a263ebe60d8d3af/assets/js/atomic/blocks/product-elements/button/index.tsx#L17

What do you think @albarin? Does it make sense?

Reply:
Hmm I see, I was just working on [fixing color values in that pattern](https://github.com/woocommerce/woocommerce-blocks/pull/10362), but I think it's better to remove them then, that way we make it also more in the wireframed line. I'll work on it. 👍